perfkitbenchmarker/providers/azure/azure_blob_storage.py

```diff
@@ -38,38 +39,34 @@ class AzureBlobStorageService(object_storage_service.ObjectStorageService):
   STORAGE_NAME = providers.AZURE
 
   def PrepareService(self, location):
-    self.storage_account = 'pkb%s' % FLAGS.run_uri
-    vm_util.IssueCommand(
-        ['azure', 'storage', 'account', 'create',
-         '--type', 'ZRS',
-         '-l', location or DEFAULT_AZURE_REGION,
-         self.storage_account])
-
-    output, _, _ = vm_util.IssueCommand(
-        ['azure', 'storage', 'account',
-         'keys', 'list', self.storage_account])
-
-    self.azure_key = regex_util.ExtractExactlyOneMatch(
-        r'Primary:* (.+)', output)
+    prefix = 'pkb%sabs' % FLAGS.run_uri
```

Comment:
abs is my abbreviation for "Azure Blob Storage". I've been trying to give my accounts semi-semantic names. (Semi because the names are very limited.)